gcc/rust/resolve/rust-ast-resolve.cc

```diff
@@ -608,6 +613,19 @@ ResolvePath::resolve_path (AST::PathInExpression *expr)
 	  return;
 	}
     }
+
+  // its fully resolved lets mark it as such
+  if (resolved_node != UNKNOWN_NODEID)
+    {
```

Comment:
nice

Reply:
As I don't need to touch this method, I've simply reverted the .h->.cc move here.